CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.11.3](https://github.com/go-gitea/gitea/releases/tag/v1.11.3) - 2020-03-10
+
+* BUGFIXES
+  * Prevent panic in stopwatch (#10670) (#10673)
+  * Fix bug on pull view when required status check no ci result (#10648) (#10651)
```

Comment:
```suggestion
  * Fix bug on pull view when required status check no ci result (#10648) (#10651)
  * Pin xgo version to go1.13.
```

Reply:
Similar to the other changelog, perhaps we should mention "explicitly building with Go 1.13"?

Reply:
@jolheiser please send your suggestion.

Reply:
Part of me wants to put it above the changelog since this isn't actually associated with a PR, but I think it might make more sense in this case to leave it in the changelog.
```suggestion
  * Fix bug on pull view when required status check no ci result (#10648) (#10651)
  * Build explicitly with Go 1.13
```